### DISCUSSION TITLE
l10s pro ultra heat not able to go back to dock (likely defect)

### DISCUSSION THREAD

**Discussion by rzetzsche** [Hypfer/Valetudo#2089] [Q&A / Support] [answered]
### Are you already using Valetudo?

Yes, for a long time

### Your questions

I got my l10s pro ultra heat today and was really excited to use it with valetudo again (I've had a S50 with valetudo for over 3 years now). Unfortunately, I got a bit too excited and flashed the robot directly instead of testing it with the stock firmware (me dumbass) and noticed that the robot is never able to return to the docking station. I think it's a problem with the base station not sending the IR signal. I will probably need to reinstall the software before sending it back (and test if the error persists with stock).

I've already read a couple of tickets with similar problems. They should all open a ticket on https://vaers.dontvacuum.me/otrs/customer.pl but I can't register there, I just never get an email back and I've tried 3 different email addresses.

It would be appreciated if someone could help.

**Comment by Hypfer** [comment, marked as answer]
Don't worry, your L10s pro ultra heat is likely fine and not defective.

While you _really_ should've tested it before rooting it, with that robot in particular, there seems to have been a breaking change in the communication between the linux-side and the mcu-side of the firmware between the factory firmware version and the version we install when rooting.
As the fastboot rooting method only updates the linux-side firmware, this mismatch causes the L10s pro ultra heat to fail to dock.

To fix that, just put the robot in the dock by hand, connect via ssh and flash another firmware build built for manual install via ssh as described here: https://valetudo.cloud/pages/usage/firmware-updates.html
That will go through the regular firmware update path and with that also bring the mcu firmware to the matching version

**Reply by rzetzsche** [reply]
This fixed the issue. Thank you so much!

**Reply by n15c** [reply]
I tested it before, so you can't blame for not doing it :)

Thanks for this hint, indeed this solved the problem. 

@Hypfer could you check the mail sending of the ticketing system? It tested multiple addresses but never got an registration mail

**Reply by duviful** [reply]
I can confirm. I had the same problem today after the fastboot rooting. Reflashing from SSH solved the problem.
Thanks!

**Reply by JBarrskog** [reply]
I will also chime in that this solved my issue!
